README.md

```diff
@@ -50,10 +50,11 @@ In order to run the test suite of the project:
 redis-server
 ```
 
-2. Make sure that `REDIS_URI` is set and points to the current Redis instance (assuming Redis is runnning on the default port `6379`):
+2. Make sure that the required environment variables are set (the following example assumes that Redis is runnning on the default port `6379`):
 
 ```bash
 export REDIS_URI=redis://localhost:6379
+export TRANSACTION_SERVICE_AUTH_TOKEN=some_other_random_token
```

Comment:
Maybe it is confusing that we add `TRANSACTION_SERVICE_AUTH_TOKEN` env var to the list of required env vars , while we don't really `panic` if it's not present: 
https://github.com/gnosis/safe-client-gateway/blob/a24617bc68cdbe1af978e81d0d649b8f03be799e/src/config/mod.rs#L36

Reply:
This is just for the test setup since the tests require this specific token.